ask-sdk-core/lib/delegation/DelegationData.ts

```diff
@@ -0,0 +1,19 @@
+import { Intent, Slot } from "ask-sdk-model";
+
+export interface DelegationData {
+    delegateTarget: DelegateTarget;
+    intentData? : Intent;
```

Comment:
Also this type needs to be extended to include samples as well that will be needed to generate ACDL utterances.